Onboarding System (#844)
italia/eid-wallet-it-docs: docs/en/credential-issuance-high-level.rst

The high-level flow begins with the User who wants to obtain a PID and starts his/her Wallet Instance (Step 0). Below the description of the steps represented in the previous picture:

>     1. **PID Provider Discovery and Trust**: the Wallet Instance discovers the trusted PID Provider using the Digital Credential Catalog and Federation Services, establishing the trust to the PID Provider according to the Trust Model and obtaining its metadata that discloses the formats of the PID, the algorithms supported, and any other parameter required for interoperability needs.

Reviewer: ```suggestion
    1. **PID Provider Discovery and Trust**: the Wallet Instance discovers the trusted PID Provider using the Digital Credential Catalog and Federation Services, establishing the trust to the PID Provider according to the Trust framework and obtaining its metadata that discloses the PID configurations, the algorithms supported, and any other required parameter.
```
